### ISSUE TITLE
Grapher omits decimals that happen to be zero, even when that decimal is a significant figure

### ISSUE THREAD

**Issue by pabloarosado** [owid/owid-grapher#1931]
## Description

Grapher omits decimals that happen to be zero, even when that decimal is a significant figure. For example, grapher shows 1.71 as "1.71", however, it shows 1.70 as "1.7". Discussed in [this Slack thread](https://owid.slack.com/archives/C46U9LXRR/p1676456180400819).

## Expected behaviour

The correct figure to show is "1.70", because, assuming this value has 3 significant figures, number 0 is just like any other number.

## Steps to reproduce

1. Go to any chart, for example [this one](https://ourworldindata.org/explorers/food-footprints?facet=none&hideControls=false&Commodity+or+Specific+Food+Product=Commodity&Environmental+Impact=Carbon+footprint&Kilogram+%2F+Protein+%2F+Calories=Per+kilogram&By+stage+of+supply+chain=false&country=Bananas~Beef+%28beef+herd%29~Beef+%28dairy+herd%29~Cheese~Eggs~Lamb+%26+Mutton~Milk~Maize~Nuts~Pig+Meat~Peas~Potatoes~Poultry+Meat~Rice~Tomatoes~Wheat+%26+Rye~Tofu+%28soybeans%29~Prawns+%28farmed%29).
2. See that some values that have a zero on the right of the decimal comma are not shown (e.g. Maize shows `1.7` instead of `1.70`).

## Screenshots

![Screen Shot 2023-02-15 at 11 13 53](https://user-images.githubusercontent.com/12246978/219125906-2764ab45-92fb-4c4f-a197-4ad393329a99.png)
![Screen Shot 2023-02-15 at 11 03 41](https://user-images.githubusercontent.com/12246978/219125909-647689dc-8ea4-4dc0-b780-57e6a6e4d8e0.png)

## Environment

Desktop:
-   OS: macOS Big Sur 11.6.4
-   Browser: chrome
-   Version: 109.0.5414.119 (Official Build) (arm64)

**Comment by marcelgerber** [comment]
Related to #1851, which also considers using the notion of significant figures.

**Comment by ikesau** [comment]
Seems like we still need a bit of discussion in #1851 to decide how extensive an overhaul (if any) our approach to rounding needs to be.

In the meantime, I'll add that this _may_ be as simple as changing some default values for numeric columns' implementations of `formatValue` wrt `trailingZeroes` being `false` by default, but I haven't actually tested that. As I discovered when I first started, tweaking this function touches a _lot_ of different figures throughout our site and requires extensive testing and QA, so I'm not sure if we'll get around to this any time soon. Maybe a good issue for Sofia or Christian to look into (or a terrible one 😅)
